src/metrics/counter/model.js

```diff
@@ -0,0 +1,32 @@
+import Joi, { joiToMongoose } from 'utils/joi';
+import mongoose from 'mongoose';
+
+const joiCounterSchema = Joi.object({
+  key: Joi.string()
+    .required()
+    .meta({ unique: true }),
+  count: Joi.number().default(0),
+});
+
+const CounterSchema = joiToMongoose(joiCounterSchema);
+
```

Comment:
please add some `get` api to view counters value

Reply:
Done https://dev.wir.by/api/metrics/counter/game-fortune-ny2021